frontend/src/routes/Search.tsx

```diff
@@ -523,13 +522,7 @@ const SearchEvent: React.FC<SearchEventProps> = ({
                         whiteSpace: "nowrap",
                         padding: 3,
                     }}>
-                        {t("video.part-of-series") + ": "}
-                        <Link to={`/!s/${keyOfId(seriesId)}`} css={{
-                            borderRadius: 4,
-                            outlineOffset: 1,
-                            position: "relative",
-                            zIndex: 4,
-                        }}>{seriesTitle}</Link>
+                        <PartOfSeriesLink {...{ seriesTitle, seriesId }} />
                     </div>}
```

Comment:
It seems like the parent div (with the `textOverflow: "ellipsis"`) also moved to `PartOfSeriesLink`, so I think that can be removed here then.